jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -571,6 +571,17 @@ public ClassLoader getClassLoader() {
 	return classLoader;
 }
 
+/**
+ * Returns the classloader used to load the receiver's class.
+ * Returns null if the class was loaded by the bootstrap (system) class loader.
+ * This skips security checks.
+ * @return		the receiver's class loader or null
```

Comment:
These big indents between the `@return` tag and the text are unnecessary.  I know they pervade the existing code but they're not something we should be propagating.

Reply:
Fixed.